packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -135,16 +137,17 @@ export class LogFileSelector
     if (logInfos.length > 0) {
       const logItems = logInfos.map(logInfo => {
         const icon = '$(file-text) ';
+        const localUTCDate = new Date(logInfo.StartTime);
+        const localDateFormatted = localUTCDate.toLocaleDateString(
```

Comment:
This is used for listing the available debug log records when running `SFDX: Get Debug Logs`
![debug_log_list](https://user-images.githubusercontent.com/1041842/65006323-92803000-d8b7-11e9-8d19-eefccc12ba98.png)